testbot_description/CMakeLists.txt

```diff
@@ -0,0 +1,155 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(testbot_description)
+
+find_package(catkin REQUIRED COMPONENTS
+  catkin
+  urdf
+  std_msgs
+  geometry_msgs
+  sensor_msgs
+  message_generation
+  roscpp
+  rospy
+  tf
+)
+
+## System dependencies are found with CMake's conventions
+# find_package(Boost REQUIRED COMPONENTS system)
+
+################################################
+## Declare ROS messages, services and actions ##
+################################################
+
+## Generate messages in the 'msg' folder
+##add_message_files(
+##  FILES
+##  JointStateExample.msg
+##)
+
+## Generate services in the 'srv' folder
+# add_service_files(
+#   FILES
+#   Service1.srv
+#   Service2.srv
+# )
+
+## Generate actions in the 'action' folder
+# add_action_files(
+#   FILES
+#   Action1.action
+#   Action2.action
+# )
```

Comment:
All this commented boilerplate should be removed, as it does not add anything and could confuse new users that want to modify this file (i.e. bad for maintainability).